moveit_simple/src/robot.cpp

```diff
@@ -1388,6 +1443,17 @@ std::map<size_t, double> Robot::getIKSeedStateFractions() const
   return ik_seed_state_fractions_;
 }
 
+bool Robot::setIKSeedStateMidPoint(const std::vector<double>& ik_seed_state_mid_point)
```

Comment:
This is great. Maybe we should combine this with setIKSeedStateFractions so we can guarantee that these always match.

Reply:
I think it is reasonable to expect the user to configure these correctly

Reply:
I added a method for checking the length of the mid_point ik joint state